### PR TITLE
fix(react-tabster): fix styles in getFocusOutlineStyles()

### PR DESCRIPTION
**Pull request by layershifter**
This PR fixes styles in `getFocusOutlineStyles()` function. Check comments on files for more details.

### CODE REVIEW

File: packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts [hunk 1]
```diff
@@ -21,7 +22,7 @@ export type FocusOutlineStyleOptions = {
  * @param options - Configures the style of the focus outline
  * @returns focus outline styles object
  */
-const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
+const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle => {
```
Comment (layershifter) [inline packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts:25]: This forcefully enables type-checking that didn't happen before for unknown reason.

File: packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts [hunk 2]
```diff
@@ -30,18 +31,17 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
...
       content: '""',
       position: 'absolute',
       pointerEvents: 'none',
-      boxSizing: 'outline-box',
       zIndex: 1,
 
-      borderStyle: 'solid',
-      borderWidth: outlineWidth,
-      borderRadius: outlineRadius,
-      borderColor: outlineColor,
```
Comment (layershifter) [inline packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts:44]: We should use shorthand functions. CSS shorthands are not support by Griffel and will be stripped out from insertion with next update.

File: packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts [hunk 3]
```diff
@@ -63,7 +63,6 @@ const defaultOptions: CreateFocusIndicatorStyleRuleOptions = {
  * NOTE: The element with the focus outline needs to have `position: relative` so that the
  * pseudo element can be properly positioned.
  *
- * @param theme - Theme used in @see makeStyles
```
Comment (layershifter) [inline packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts:66]: This option does not exist.

File: packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts [hunk 4]
```diff
@@ -89,6 +88,7 @@ export const createFocusOutlineStyle = (
  * Should be used only when @see createFocusOutlineStyle does not fit requirements
  *
  * @param style - styling applied on focus, defaults to @see getDefaultFocusOutlineStyes
+ * @param options - Configure the style of the focus outline
```
Comment (layershifter) [inline packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts:91]: Missing description for the param.

File: packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts [hunk 5]
```diff
@@ -30,18 +31,17 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
   const outlineOffsetRight = (outlineOffset as FocusOutlineOffset)?.right || outlineOffset;
 
   return {
-    borderColor: 'transparent',
+    ...shorthands.borderColor('transparent'),
     ':after': {
       content: '""',
       position: 'absolute',
       pointerEvents: 'none',
-      boxSizing: 'outline-box',
```
Comment (layershifter) [inline packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts:38]: `outline-box` is not supported value

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                                                                    Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |   `73.035 kB`<br />`22.11 kB` |  `74.589 kB`<br />`22.458 kB` | `1.554 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`348 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `172.513 kB`<br />`48.34 kB` | `175.621 kB`<br />`48.963 kB` | `3.108 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`623 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `103.976 kB`<br />`31.998 kB` |  `105.53 kB`<br />`32.333 kB` | `1.554 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`335 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `106.331 kB`<br />`32.361 kB` | `107.885 kB`<br />`32.695 kB` | `1.554 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`334 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `23.351 kB`<br />`7.894 kB` |   `25.402 kB`<br />`8.251 kB` | `2.051 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`357 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `22.598 kB`<br />`7.642 kB` |   `24.245 kB`<br />`7.986 kB` | `1.647 kB` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`344 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                  |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                         |  `28.013 kB`<br />`8.059 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                         |  `33.336 kB`<br />`9.037 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                 |  `29.763 kB`<br />`8.656 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                               |  `36.235 kB`<br />`9.857 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                             |   `37.395 kB`<br />`8.68 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                     |  `53.205 kB`<br />`15.27 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                               | `48.898 kB`<br />`14.083 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                   |   `7.653 kB`<br />`3.246 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                   |   `8.931 kB`<br />`3.689 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                 |   `7.626 kB`<br />`3.272 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>                                                   |   `6.813 kB`<br />`2.895 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `32.526 kB`<br />`10.645 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                              |  `11.102 kB`<br />`4.504 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                      | `96.776 kB`<br />`29.553 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                         |   `6.267 kB`<br />`2.168 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                       |   `14.009 kB`<br />`5.25 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                      |  `42.76 kB`<br />`14.701 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6919440703c22aba846cd1f92da2fcb0de85d81c'>6919440703c22aba846cd1f92da2fcb0de85d81c</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/Avatar-mount_base.html">1124</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/Avatar-mount.html">1090</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/Button-mount_base.html">666</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/Button-mount.html">628</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProvider-mount_base.html">2158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProvider-mount.html">2136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-mount_base.html">311</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-mount.html">308</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">273</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">271</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">346</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/MakeStyles-mount_base.html">1800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22070/perf-test-react-components/MakeStyles-mount.html">1890</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by ling1726 — APPROVED** [review]

**Pull request merged**